### PR TITLE
Adds "LiT: Zero-Shot Transfer with Locked-image text Tuning".

### PR DESCRIPTION
**Pull request by andsteing**
Paper: https://arxiv.org/abs/2111.07991

Co-authored-by: Alexander Kolesnikov <akolesnikov@google.com>
Co-authored-by: Lucas Beyer <lbeyer@google.com>
Co-authored-by: Xiaohua Zhai <xzhai@google.com>

### CODE REVIEW

**Comment by andsteing** [comment]
### Previewing the README

https://github.com/andsteing/big_vision/blob/lit/big_vision/configs/proj/image_text/README.md

### Testing the Colab

https://colab.research.google.com/github/andsteing/big_vision/blob/lit/big_vision/configs/proj/image_text/lit.ipynb

and then modify 
```
!git clone --branch=main --depth=1 https://github.com/google-research/big_vision
```
to
```
!git clone --branch=lit --depth=1 https://github.com/andsteing/big_vision
```

File: big_vision/configs/proj/distill/bigsweep_pets.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Big Vision Authors.
```
Comment (lucasb-eyer) [inline big_vision/configs/proj/distill/bigsweep_pets.py:1]: This file is here by mistake. It doesn't exist anymore internally, so this makes me think that maybe the PR is based on a somewhat stale version? Could you double-check?

File: big_vision/configs/proj/image_text/lit_coco.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      'pool_type': 'tok',
+      'head_zeroinit': False,
+  })
+  config.model.text = ConfigDict({
+      'config': txt_name,
+      'head_zeroinit': False,
+  })
+  config.model.temperature_init = 10.0
+  dim = {'B': 768, 'L': 1024}[arg.img[0]]
+  config.model.out_dim = (dim if arg.img_head else None, dim)  # (image_out_dim, text_out_dim)
```
Comment (lucasb-eyer) [inline big_vision/configs/proj/image_text/lit_coco.py:114]: Nice :)

**Review by lucasb-eyer — APPROVED** [review]
Thanks! Since we already reviewed most of it internally, I only skimmed that things still look good after open-sourcing. Looks like one obsolete config sneaked in (see other comment), maybe double-check why/how, remove it, and then we're good to go.

File: big_vision/configs/proj/distill/bigsweep_pets.py (hunk 1 above)
Comment (andsteing) [reply big_vision/configs/proj/distill/bigsweep_pets.py]: ah right ... the `rsync` was missing the `--del` flag; thanks for the catch!

**Pull request merged**